test/libsolidity/syntaxTests/constants/constant_override.sol

```diff
@@ -0,0 +1,3 @@
+uint constant override x;
+// ----
+// TypeError 4266: (0-24): Uninitialized "constant" variable.
```

Comment:
This test should maybe initialize it? Does that error out? This is not the error I would expect in this test in any case.

Reply:
Good catch!